lib/mayaUsd/ufe/CMakeLists.txt

```diff
@@ -235,12 +248,17 @@ if(CodeWrapperHandler IN_LIST UFE_PREVIEW_FEATURES)
     )
 endif()
 
+if(UFE_CAMERAHANDLER_HAS_FINDALL)
+    list(APPEND HEADERS
+        ProxyShapeCameraHandler.h
```

Comment:
This seems to be causing an issue for Maya 2023 platform when cameraHandler_v3_4::findAll isn't present.

`Global.cpp(74): fatal error C1083: Cannot open include file: 'mayaUsd/ufe/ProxyShapeCameraHandler.h': No such file or directory`